plugins/filters/add_prefix.py

```diff
@@ -0,0 +1,17 @@
+from ansible import utils
+
+
+def add_prefix(value, prefix=''):
+    """Add string prefix to item.
+
+    {{ ['hanukkah', 'easter', 'pessah'] | map('add_prefix', 'happy ' }}
+    -> ['happy hanukkah', 'happy easter', 'happy pessah']
+    """
+
+    return prefix + value
+
+
+class FilterModule(object):
+
+    def filters(self):
+        return {'add_prefix': add_prefix}
```

Comment:
Please keep the convention same as other filters:

return {
              'add_prefix': add_prefix
}